### ISSUE TITLE
Error in loading VFs

### ISSUE THREAD

**Issue by yanone** [simoncozens/babelfont#30]
I want to start using babelfont in fontbakery checks, because it makes access to data so much easier. (Not sure yet whether it'll be accepted because of larger dependency tree)

However, one error I found when loading variable fonts is this:

```
 >> com.google.fonts/check/legacy_accents
    Check that legacy accents aren't used in composite glyphs. 
    with ofl/ysabeau/Ysabeau[wght].ttf

      Rationale:                                                                
                                                                                
      Legacy accents should not be used in accented glyphs. The use of legacy   
      accents in accented glyphs breaks the mark to mark combining feature that 
      allows a font to stack diacritics over one glyph.                         
                                                                                
      Use combining marks instead as a component in composite glyphs. Legacy    
      accents should not have anchors and positive width.                       
                                                                                
    More info: https://github.com/googlefonts/fontbakery/issues/3959


    ERROR Failed with TypeError: cannot unpack non-iterable                     
          float object                                                          
          ↳ File                                                                
          "/Users/yanone/.pyenv/versions/3.11.1/Library/Frameworks/Python.framew
          ork/Versions/3.11/lib/python3.11/site-packages/fontbakery/checkrunner.
          py", line 194, in _exec_check                                         
          for sub_result in result:  # Might raise.                             
          ↳ File                                                                
          "/Users/yanone/.pyenv/versions/3.11.1/Library/Frameworks/Python.framew
          ork/Versions/3.11/lib/python3.11/site-packages/fontbakery/profiles/uni
          versal.py", line 619, in com_google_fonts_check_legacy_accents        
          layer = font.default_master.get_glyph_layer(glyph.name)               
          ^^^^^^^^^^^^^^^^^^^                                                   
          ↳ File                                                                
          "/Users/yanone/.pyenv/versions/3.11.1/Library/Frameworks/Python.framew
          ork/Versions/3.11/lib/python3.11/functools.py", line 1001, in get     
          val = self.func(instance)                                             
          ^^^^^^^^^^^^^^^^^^^                                                   
          ↳ File                                                                
          "/Users/yanone/.pyenv/versions/3.11.1/Library/Frameworks/Python.framew
          ork/Versions/3.11/lib/python3.11/site-packages/babelfont/Font.py",    
          line 140, in default_master                                           
          default_loc = {a.tag: a.map_forward(a.default) for a in self.axes}    
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                  
          ↳ File                                                                
          "/Users/yanone/.pyenv/versions/3.11.1/Library/Frameworks/Python.framew
          ork/Versions/3.11/lib/python3.11/site-packages/babelfont/Font.py",    
          line 140, in <dictcomp>                                               
          default_loc = {a.tag: a.map_forward(a.default) for a in self.axes}    
          ^^^^^^^^^^^^^^^^^^^^^^^^                                              
          ↳ File                                                                
          "/Users/yanone/.pyenv/versions/3.11.1/Library/Frameworks/Python.framew
          ork/Versions/3.11/lib/python3.11/site-packages/babelfont/Axis.py",    
          line 110, in map_forward                                              
          return piecewiseLinearMap(v, {k: v for k, v in self.map})             
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^                                           
          ↳ File                                                                
          "/Users/yanone/.pyenv/versions/3.11.1/Library/Frameworks/Python.framew
          ork/Versions/3.11/lib/python3.11/site-packages/babelfont/Axis.py",    
          line 110, in <dictcomp>                                               
          return piecewiseLinearMap(v, {k: v for k, v in self.map})             
          ^^^^
```

The error occurs when accessing the default master like so `font.default_master.get_glyph_layer(glyph.name)`

(The check isn't committed to Fontbakery yet)

**Comment by yanone** [comment]
Actually, Babelfont is already part of Fontbakery as part of a gftools (among others) dependency, so I'd like to go ahead with this approach

**Comment by simoncozens** [comment]
fixed by #32
